temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -412,6 +421,18 @@ public Builder setStaticDetails(String staticDetails) {
       return this;
     }
 
+    /** Request ID. */
```

Comment:
In Go we got away with not exposing this.
I would want us to eventually expose it, it's pretty critical for deduping requests that are retried by an external caller.
Without this if the workflow started completes between retries, it will be restarted.

Reply:
Documented